### ISSUE TITLE
can't refactor python code

### ISSUE THREAD

**Issue by droopy4096** [lgeiger/ide-python#86]
having installed ide-python, atom-ide-ui and pythonl-language-server etc. I get no options in atom for code refactoring, like rename a class etc. Invoking command pallete doesn't reveal a command for refactor.... 

python-language-server (0.15.1)
atom-1.23.3
atom-ide-ui-0.9.1
ide-python-0.8.0

**Comment by exploide** [comment]
Yes, this is true. But `ide-python` cannot do something against this as it is an upstream issue of `atom-languageclient` and `atom-ide-ui` not supporting this yet.

https://github.com/atom/atom-languageclient/issues/13
https://github.com/facebook-atom/atom-ide-ui/issues/145

**Comment by mwielondek** [comment]
Seeing that the issue in `atom-languageclient` is now closed, and you're no longer relying on `atom-ide-ui`, does it put this issue in a new light?

**Comment by aminya** [comment]
@ThatXliner See this 
https://github.com/atom-community/atom-languageclient/pull/141

**Comment by ThatXliner** [comment]
Cool. Seems like a good place to contribute for this feature
